docs/deployment/settings.md

```diff
@@ -438,6 +438,7 @@ kyuubi.operation.plan.only.output.style|plain|Configures the planOnly output sty
 kyuubi.operation.progress.enabled|false|Whether to enable the operation progress. When true, the operation progress will be returned in `GetOperationStatus`.|boolean|1.6.0
 kyuubi.operation.query.timeout|&lt;undefined&gt;|Timeout for query executions at server-side, take affect with client-side timeout(`java.sql.Statement.setQueryTimeout`) together, a running query will be cancelled automatically if timeout. It's off by default, which means only client-side take fully control whether the query should timeout or not. If set, client-side timeout capped at this point. To cancel the queries right away without waiting task to finish, consider enabling kyuubi.operation.interrupt.on.cancel together.|duration|1.2.0
 kyuubi.operation.result.codec|simple|Specify the result codec, available configs are: <ul> <li>SIMPLE: the result will convert to TRow at the engine driver side. </li> <li>ARROW: the result will be encoded as Arrow at the executor side before collecting by the driver, and deserialized at the client side. note that it only takes effect for kyuubi-hive-jdbc clients now.</li></ul>|string|1.7.0
+kyuubi.operation.result.compression.codec|lz4|The codec used to compress arrow-based result. By default, Kyuubi provides three codecs: `lz4`, `zstd`, `gzip`|string|1.7.0
```

Comment:
As `no compression` is also supported in this PR, the `none` value can be added to doc in a way to disable compression.